### ISSUE TITLE
LowercasePHPFunctions sniff failing on "new \File()"

### ISSUE THREAD

**Issue by josephzidell** [squizlabs/PHP_CodeSniffer#2452]
I've got a class `File` in the global namespace, and with the latest update, PHPCS is showing this false positive:

```
Calls to PHP native functions must be lowercase; expected "file" but found "File"
```

Code:
```php
$filePath = new \File($path);
```

**Comment by gsherwood** [comment]
Looks to be caused by #2252 but is a quick fix

**Comment by gsherwood** [comment]
Fix will be in 3.4.2, which I'll be trying to release next week

**Comment by gsherwood** [comment]
Thanks a lot for reporting this

**Comment by josephzidell** [comment]
Thanks for the quick response and fix!
